styles.css

```diff
@@ -0,0 +1,60 @@
+
+/* ......... the header section.......... */
+header {
+  width: 375px;
+  height: 48px;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-around;
+  color: #fff;
+  padding-top: 44px;
+}
+
+.name {
+  font-family: 'Roboto', sans-serif;
+  line-height: 24px;
+}
+
+/* ............headline section............. */
+
+.headline {
+  background: #1c1a19;
+  color: #fff;
+  padding-bottom: 80px;
+  padding-top: 80px;
```

Comment:
- [ ] Kindly add the background-image missing 

![image](https://user-images.githubusercontent.com/93630700/205456497-f267ef4b-48ca-4ea6-b560-111a17b48bdc.png)